src/pages/gen1/[platform]/prev/build-ui/index.mdx

```diff
@@ -0,0 +1,49 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Build UI',
+  description: 'Build UI',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'android',
+        'angular',
+        'flutter',
+        'javascript',
+        'nextjs',
+        'react',
+        'react-native',
+        'swift',
+        'vue'
+      ],
+      canonicalPath: '/gen1/react/prev/build-ui/'
+    }
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Note: This page is exclusive to the latest version of our documentation. To ensure you have access to the most current information and features, please make sure you're viewing the latest version by using the version switcher in the left menu.
+
```

Comment:
Any feedback on this? Should it be worded differently? Should I add it in a callout instead?